src/main/java/org/opensearch/sdk/SDKClient.java

```diff
@@ -74,6 +70,17 @@ public TlsDetails create(final SSLEngine sslEngine) {
                 throw new RuntimeException(e);
             }
         });
+        return builder;
+    }
+
+    /**
+     * Creates OpenSearchClient for SDK. It also creates a restClient as a wrapper around Java OpenSearchClient
+     * @param hostAddress The address of OpenSearch cluster, client can connect to
+     * @param port The port of OpenSearch cluster
+     * @return SDKClient which is internally an OpenSearchClient. The user is responsible for calling {@link #doCloseRestClient()} when finished with the client
```

Comment:
I'm not sure the words "SDKClient which is..." here and for the HLRC are right, as we are inside the SDKClient class so it's not really an SDKClient.  Would prefer words such as "The SDKClient implementation of ..."